pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -14,8 +17,12 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-func ProvideService(cfg *setting.Cfg, store accesscontrol.Store, routeRegister routing.RouteRegister) (*Service, error) {
-	service := ProvideOSSService(cfg, store)
+const (
+	cacheTTL = 10 * time.Second
```

Comment:
thought: I wonder if we've documented this time anywhere. I can't find anything related to it, but mentioning this in case you know of any docs that would need to be updated.